Azure/Dev/resource_virtual_network.tf

```diff
@@ -184,7 +184,8 @@ resource "azurerm_application_gateway" "AppGate" {
       redirect_configuration,      
       url_path_map,     
       tags, # AGIC adds tags which need to be ignored. Can't seem to ignore the individual tags
-      ssl_certificate # see issue above 
+      ssl_certificate, # see issue above
+      waf_configuration.0 # see issue above
```

Comment:
can we please not ignore the waf rules, just change the tf to have them in the same order as they are in azure?

Reply:
@stuart-bass-bjss Unfortunately, I tried that and set them to the same order as it is in Azure but that did not resolve the issue as it still reports them needing removing/re-adding - Terraform wants to 'own' the ordering which can only be done with an import or an apply. Once the change is made to the Azure TF module this line can be removed alongside the SSL one and an apply performed that will re-apply these settings exactly as they are but owned by Terraform.

Reply:
I've actually had another look and am applying the changes now